release/releaser.go

```diff
@@ -203,8 +203,10 @@ func logEvent(inst *instance.Instance, executeErr error, release flux.Release) e
 	}
 
 	var serviceIDs []flux.ServiceID
-	for _, id := range release.Result.ServiceIDs() {
-		serviceIDs = append(serviceIDs, flux.ServiceID(id))
+	for k, v := range release.Result {
```

Comment:
Gotcha, ta. If things are skipped it's because they were _supposed_ to be updated (e.g., they were specifically requested) but weren't in fact updated. It's then a question of reporting what the user expects to see (the things they mentioned) vs being accurate (what was *actually* updated).

Let's preserve the current behaviour for now. When we include the result in the notification this will be resolved.